src/openbook/database.py

```diff
@@ -0,0 +1,26 @@
+from collections.abc import Generator
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import Session, sessionmaker
+
+from openbook.models.orm import Base
+
+DATABASE_URL = "sqlite:///mydb.db"
```

Comment:
This should be configurable via environment variables, or whatever.

<https://docs.pydantic.dev/latest/concepts/pydantic_settings/>